Jenkinsfile.s390x.its

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+pipeline {
+  agent { node { 's390x' } }
+  options {
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '5'))
+    timeout(time: 180, unit: 'MINUTES')
+  }
+  parameters {
+    string( defaultValue: 'master', description: 'Core Its branch (default master)',
+            name: 'ITS_BRANCH' )
+  }
+  stages {
+    stage("Build Maven Core") {
+      steps {
+        withEnv(["JAVA_HOME=${ tool "ibm-semeru-8u362-b09" }", "PATH+MAVEN=${tool 'Maven 3.6.3'}/bin:${env.JAVA_HOME}/bin"]) {
```

Comment:
Is 3.6.3 what we're really using? Looks like other ITs are somewhere in 3.8 right now

Reply:
You are right.  I see on the main Jenkinsfile it should be 3.8.x.

We won't need this new `Jenkinsfile.s390x.its` file if s390x builds are run nightly or for commits only (no PR checks) (Please see https://github.com/apache/maven/pull/1239#issuecomment-1719681339).

If that's ok with you then I can either a) create a new PR or b) reuse this PR to reinstate the Jenkinsfile.s390x that was previously removed.  Will need help with disabling PR checks in https://ci-maven.apache.org/job/Maven/job/Maven%20Core%20s390x/ as I have no access to Jenkins configuration.  Please let me know.  Thanks!

Reply:
@elharo, your thoughts on adding `Jenkinsfile.s390x` back and disable PR checks on s390x?  (Please see https://github.com/apache/maven/pull/1239#issuecomment-1719681339).

Reply:
I don't have any particular opinion about that. As long as tests are passing, I'm OK with whatever setup works best. I just want to avoid the situation we got into earlier where the S390x build was broken and blocking work, and no one was addressing it. I don't think anyone here not from IBM has the resources to debug an S390 specific problem. We have enough trouble figuring out what's going on when a test fails only on Windows. I'm also leery of IT failures that happen because test setup or verification isn't configured properly for some particular platform, rather than an actual bug in the maven product. I think we've all burned a lot of hours debugging those kinds of "failures".

Reply:
the job here has been configured to run only on `master` https://ci-maven.apache.org/job/Maven/job/Maven%20Core%20s390x/
by the way gh checks were disable. 
so this job was not doing any spam except the person registered to received emails. And he asked to do so.

Reply:
Thanks @olamy.  Does this mean I can close this PR for `Jenkinsfile.s390x.its` as PR checks won't run on s390x?  

I will create another PR to bring back `Jenkinsfile.s390x`.